### ISSUE TITLE
In workflows Liquid tags are not resolved.

### ISSUE THREAD

**Issue by giannik** [OrchardCMS/OrchardCore#8860]
After upgrading to latest dev with fluid 2 my workflows are not resolving liquid tags.
To repro: 

1. Create a form to trigger workflow 
2. In the workflow use the CreateContentTask activity to create a content item with liquid expressions like :
```
{

  "ContentType": "LearnerInterestFormResponse",
  "Owner": "{{Request.Form.touserid}}",
  "Author": "{{Request.Form.fromusername}}",
  "MyContentType": {
    
    "Message": {
      "Text": "{{Request.Form.message}}"
    }
   }
 }
```

3.trigger the workflow from the form 
4. The contentitem created has empty values for the liquid placeholders.

Even putting a breakpoint in 
https://github.com/OrchardCMS/OrchardCore/blob/1f581a14a1f37d0105d93331a83d87e3c4bc310a/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/OrchardCoreBuilderExtensions.cs#L83

does not get caught.

**Comment by hishamco** [comment]
@sebastienros is this related to upgrading to Fluid 2.0?

**Comment by Skrypt** [comment]
Yes it is.

**Comment by jtkech** [comment]
I think it has been fixed through #8926

Feel free to re-open the issue if it still doen't work
